operations/research-and-development/README.md

```diff
@@ -125,15 +153,15 @@ R&D includes engineering, QA, product management, and design \(UX, UI\).
 * Ian Whitlock - Engineer
```

Comment:
For the growth team above, can you help remove Mario, and add `Najla Dadmand - Product Manager` and `Eric Nelson - Analyst`